src/example.py

```diff
@@ -0,0 +1,21 @@
+import numpy as np
+
+# Create a 2-D array, set every second element in
+# some rows and find max per row:
+
+x = np.arange(15, dtype=np.int64).reshape(3, 5)
+x[1:, ::2] = -99
+x
```

Comment:
Is this needed for this script?

Reply:
Nope. I just copy pasted the entire numpy example blindly.
Will remove after I found or write a better example